recipes/libcamera/meta.yaml

```
@@ -0,0 +1,122 @@
+{% set libcamera_version = "0.4.0" %}
+{% set libcamera_rpi_version = "0.4.0+rpt20250213" %}
+{% set rpi_url_safe_version = libcamera_rpi_version | urlencode %}
+{% set build = 0 %}
+
+{% if variant == "upstream" %}
+  # install upstream variant by dafault
+  {% set build = build + 100 %}
+{% endif %}
+
+package:
+  name: libcamera
+  version: {{ libcamera_version }}
+
+source:
+{% if variant == "upstream" %}
+  # libcamera does not offer tar balls
+  git_url: https://git.libcamera.org/libcamera/libcamera.git
+  git_rev: v{{ libcamera_version }}
+{% elif variant == "rpi_fork" %}
+  # In conversations with upstream
+  # https://github.com/raspberrypi/libcamera/issues/251
+  # We found that they attempt to maintain ABI compatibility
+  # Thus we build Raspberry-Pi specific modifications as as a variant instead of a separate package
+  url: https://github.com/raspberrypi/libcamera/releases/download/v{{ rpi_url_safe_version }}/libcamera-{{ rpi_url_safe_version }}.tar.xz
+  sha256: 764b9b2ab16dbc4e90cd858cd2019be76e087e093bbdd08048d87d0a704f8e28
+{% endif %}
+  patches:
+    - memfd_symbols.patch
+
+build:
+  number: {{ build }}
+  skip: true  # [not linux]
```

```suggestion
  skip: true  # [not linux]
  {% if variant == "rpi_fork" %}
  skip: true  # [not aarch64]
  {% endif %}
```

right???? does the rpi fork make sense for other linux?

---

no no, that's perfectly correct, thank you!